assets/prefabs/Blackberry/Blackberry.prefab

```diff
@@ -0,0 +1,10 @@
+{
+  "parent": "engine:iconItem",
+  "DisplayName": {
+    "name": "Blackberry"
+  },
+  "Item": {
+    "icon": "SimpleFarming:SimpleFarming#Blackberry",
+    "stackId": "Blackberry"
```

Comment:
Use `SimpleFarming:Blackberry` for the icon, to refer `textures/Blackberry.png` after switching from the atlas to the textures folder